cfme/tests/cloud_infra_common/test_provisioning.py

```diff
@@ -308,13 +309,26 @@ def test_provision_from_template_using_rest(appliance, request, provider, vm_nam
 
 
 VOLUME_METHOD = ("""
+clone_options = {{
```

Comment:
Optional: I think this would be more readable if it was indented like `ONE_FIELD` below. 

Question: Why are these not at the top of the file?